addons/project/views/project_views.xml

```diff
@@ -571,75 +575,85 @@
+                                                <label for="allow_task_dependencies"/>
+                                                <div class="text-muted">
+                                                    Determine the order in which to perform tasks
+                                                </div>
+                                            </div>
                                         </div>
-                                    </div>
-                                </div>
-                                <div class="col-lg-6 o_setting_box" id="task_dependencies_setting" groups="project.group_project_task_dependencies">
-                                    <div class="o_setting_left_pane">
-                                        <field name="allow_task_dependencies"/>
-                                    </div>
-                                    <div class="o_setting_right_pane">
-                                        <label for="allow_task_dependencies"/>
-                                        <div class="text-muted">
-                                            Determine the order in which to perform tasks
+                                        <div class="o_setting_box mt-4" id="project_milestone_setting" groups="project.group_project_milestone">
+                                            <div class="o_setting_left_pane">
+                                                <field name="allow_milestones"/>
+                                            </div>
+                                            <div class="o_setting_right_pane">
+                                                <label for="allow_milestones"/>
+                                                <div class="text-muted">
+                                                    Track major progress points that must be reached to achieve success
+                                                </div>
+                                            </div>
                                         </div>
                                     </div>
-                                </div>
-                                <div class="col-lg-6 o_setting_box" id="project_milestone_setting" groups="project.group_project_milestone">
-                                    <div class="o_setting_left_pane">
-                                        <field name="allow_milestones"/>
-                                    </div>
-                                    <div class="o_setting_right_pane">
-                                        <label for="allow_milestones"/>
-                                        <div class="text-muted">
-                                            Track major progress points that must be reached to achieve success
+                                </group>
+                                <group name="group_time_managment" string="Time Managment" invisible="1" class="row mt16 o_settings_container"/>
```

Comment:
To avoid making that group twice and replacing one when both are installed, I have added invisible group to the project because it will contain divs from both timesheet and planning
Thanks